### DISCUSSION TITLE
Possible to save Display, Playback and subtitles settings to user profile?

### DISCUSSION THREAD

**Discussion by shamu450** [hrfee/jfa-go#63] [Ideas]
Hello, love jfa-go! Just finished installing it last night and got it all pretty much working, a few minor issues but that's not for here. 

Please, I was wondering tho if it would be possible to add the functionality to save the Display, Playback and Subtitle settings to a user profile so we can apply them to a new user along with the Home settings. I for one would find this super helpful, I hope others would as well. 

Thank you for reading and for making a great addition to jellyfin!!

**Comment by hrfee** [comment]
Sorry it took a while to respond. As many as possible of these settings are already stored and applied by profiles. Annoyingly, quite a few of the settings are client-side and are stored locally on each device, which means they can't be changed by jfa-go.

**Comment by shamu450** [comment]
Thanks for the quick reply! I figured you tried to include as many as you
could but I had to ask to make sure. Thanks for taking the time to answer
and for creating a great plugin!
